app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = dict()
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.__class__.people[name] = self
```

Comment:
```suggestion
        self.people[name] = self
```

Reply:
Explain why if we need add to class's attribute we use instance of class not class

Reply:
sorry, my mistake, but it would be better to use Person.people